contracts/src/v0.8/Verifier.sol

```diff
@@ -56,6 +56,8 @@ contract Verifier is IVerifier, ConfirmedOwner, TypeAndVersionInterface {
     uint32 latestEpoch;
     /// The latest config digest set
     bytes32 latestConfigDigest;
+    // Whether or not the verifier for this feed has been deactivated
+    bool isDeactivated;
```

Comment:
Since this is a storage var, struct packing matters and this bool should be included with the uint32s in the same slot

Reply:
good spot